editor/plugins/canvas_item_editor_plugin.cpp

```diff
@@ -3460,9 +3453,9 @@ void CanvasItemEditor::_draw_control_helpers(Control *control) {
 }
 
 void CanvasItemEditor::_draw_selection() {
-	Ref<Texture2D> pivot_icon = get_editor_theme_icon(SNAME("EditorPivot"));
-	Ref<Texture2D> position_icon = get_editor_theme_icon(SNAME("EditorPosition"));
-	Ref<Texture2D> previous_position_icon = get_editor_theme_icon(SNAME("EditorPositionPrevious"));
+	const Ref<Texture2D> pivot_icon = get_editor_theme_icon(SNAME("EditorPivot"));
+	const Ref<Texture2D> position_icon = get_editor_theme_icon(SNAME("EditorPosition"));
+	const Ref<Texture2D> previous_position_icon = get_editor_theme_icon(SNAME("EditorPositionPrevious"));
```

Comment:
```suggestion
	Ref<Texture2D> pivot_icon = get_editor_theme_icon(SNAME("EditorPivot"));
	Ref<Texture2D> position_icon = get_editor_theme_icon(SNAME("EditorPosition"));
	Ref<Texture2D> previous_position_icon = get_editor_theme_icon(SNAME("EditorPositionPrevious"));
```
We don't forbid using `const` here but for a cleanup lets avoid unnecessary noise with changes that aren't necessary